simple_plot.py

```diff
@@ -313,11 +333,22 @@ def plotData(NQuery, input_table, FigureStrBase, variables, xMin, xMax,
         ax.text(xf + 0.05, yf-0.01, str(10**rad) + ' ' + str(zMin.unit),
                 transform=ax.transAxes)
 
+    # Saving the plots
+
+    if html_dir is None:
+        html_dir = "/"
+
+    if png_dir is None:
+        png_dir = "/"
+
+    html_dir = html_dir.rstrip("/") + "/"
+    png_dir = png_dir.rstrip("/") + "/"
+
     html = mpld3.fig_to_html(figure)
-    with open(FigureStrBase+NQuery+'.html', 'w') as f:
+    with open(html_dir+FigureStrBase+NQuery+'.html', 'w') as f:
```

Comment:
better to use `os.path.join(html_dir, FigureStrBase+NQuery...` and then avoid the rstrip + '/' stuff above